src/output_list.rs

```diff
@@ -0,0 +1,285 @@
+            eprintln!("No subcommand provided");
+            std::process::exit(1);
+        }
+    }
+}
+
+fn handle_list() {
+    let client = new_client();
+    let lists = client
+        .list_output_recipient_lists()
+        .expect("Failed to list output lists");
+
+    if lists.is_empty() {
+        println!("No output lists found");
+        return;
+    }
+
+    let mut builder = Builder::default();
+    builder.push_record(["ID", "NAME", "DESCRIPTION"]);
+
+    for list in lists {
+        let desc = list.description.as_deref().unwrap_or("-");
+        builder.push_record([
+            &list.id,
+            &list.name,
+            desc,
+        ]);
+    }
+
+    let mut table = builder.build();
+    table.with(Style::empty());
+    println!("{}", table);
+}
+
+fn handle_show(args: &ArgMatches) {
+    let name = args.get_one::<String>("name").unwrap();
+    let client = new_client();
+
+    let lists = client
+        .find_output_recipient_lists(name)
+        .expect("Failed to find output list");
+
+    if lists.is_empty() {
+        eprintln!("{} Output list '{}' not found", red!("✗"), name);
+        std::process::exit(1);
+    }
+
+    let list = &lists[0];
+
+    let outputs = client
+        .get_output_list_members(&list.id)
+        .expect("Failed to get outputs in list");
+
+    println!("ID:          {}", list.id);
+    println!("Name:        {}", list.name);
+    println!("Description: {}", list.description.as_deref().unwrap_or("-"));
+    println!("Outputs:     {}", outputs.len());
+
+    if !outputs.is_empty() {
+        println!("\nOutputs in list:");
+        let mut builder = Builder::default();
+        builder.push_record(["ID", "NAME"]);
+
+        for output in outputs {
+            builder.push_record([&output.id, &output.name]);
+        }
+
+        let mut table = builder.build();
+        table.with(Style::empty());
+        println!("{}", table);
+    }
+}
+
+fn handle_create(args: &ArgMatches) {
+    let name = args.get_one::<String>("name").unwrap();
+    let description = args.get_one::<String>("description").map(|s| s.to_string());
+    let group_name = args.get_one::<String>("group")
+        .map(|s| s.to_string())
+        .unwrap_or_else(|| "system".to_string());
+
+    let client = new_client();
+
+    let groups = client
+        .find_groups(&group_name)
+        .expect("Failed to find group");
+
+    if groups.is_empty() {
+        eprintln!("{} Group '{}' not found", red!("✗"), group_name);
+        std::process::exit(1);
+    }
+
+    let group_id = &groups[0].id;
+
+    let new_list = NewOutputRecipientList {
+        name: name.clone(),
+        description,
+        group: group_id.clone(),
+        add_outputs: Vec::new(),
+    };
+
+    let created_list = client
+        .create_output_recipient_list(new_list)
+        .expect("Failed to create output list");
+
+    println!("{} Created output list '{}' with ID {}",
+        green!("✓"), created_list.name, created_list.id);
```

Comment:
Is the green check-mark necessary? is it consistent?

This is generally true for most of this file